.github/workflows/ci-pre-commit.yml

```diff
@@ -32,6 +32,11 @@ jobs:
           profile: minimal
           toolchain: nightly-2024-03-26
           components: rustfmt, clippy
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly-2024-08-04
+          components: rustfmt, clippy
```

Comment:
did we need to use the newer version? It feels weird maintaining 3 different toolchain versions for fmt and clippy.

Reply:
yeah old version broke some things